packages/cli/src/commands/base-command.ts

```diff
@@ -71,8 +72,20 @@ export abstract class BaseCommand extends Command {
 
 	protected async loadModules() {
 		for (const moduleName of this.modulesConfig.modules) {
-			await import(`../modules/${moduleName}/${moduleName}.module`);
-			this.logger.debug(`Loaded module "${moduleName}"`);
+			const preInitModule = (await import(
+				`../modules/${moduleName}/${moduleName}.pre-init`
+			)) as ModulePreInit;
```

Comment:
If you `await import()` a filepath that doesn't exist, this returns `undefined` instead of throwing?

Reply:
it does throw, I'll catch it indeed